docker/mips64-linux-muslabi64-gcc.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# this fixes an issue of missing symbols from the command lines
+# these soft-float routines are required even for hard-float targets.
+#   (strtod.lo): undefined reference to symbol '__trunctfsf2@@GCC_3.0'
```

Comment:
Should we ln this exec to make sure it's executed? Is that maybe not really needed (since cwd _shouldn't_ change)

Reply:
The `cwd` won't change from where the script is invoked, so I don't think this is an issue.